app/models/building.rb

```diff
@@ -59,6 +59,7 @@ def additional_schema_dot_org_attributes
       email:,
       geo: coordinates,
       googleId: google_id,
+      hours:
```

Comment:
Why are we adding hours?

Reply:
@cdoyle-temple Do you want that test removed then?

Reply:
Yes, it should not have been included.